bigbluebutton-html5/imports/ui/components/actions-bar/mute-button/container.jsx

```diff
@@ -11,7 +11,7 @@ export default createContainer(() => {
   const userId = Auth.userID;
   const voiceUser = VoiceUsers.findOne({ intId: userId });
 
-  const { muted, joined, talking } = voiceUser;
+  const { muted, joined, talking, listenOnly } = voiceUser;
```

Comment:
Since we are here anyway, not sure who did it, but could you move [This](https://github.com/KDSBrowne/bigbluebutton/blob/83adf4200f3f72ed18021fc2b68a6ce3889c3a24/bigbluebutton-html5/imports/ui/components/actions-bar/mute-button/container.jsx#L11-L16) whole part to the service?
It should be 2 functions, `getVoiceUser` and `toggleSelfVoice` in the service (I would pass it with that name as well, but somebody thought that `callback` would be better 😕 ).